pymatgen/command_line/bader_caller.py

```diff
@@ -100,14 +98,14 @@ def temp_decompress(file: str | Path, target_dir: str = ".") -> str:
             """
             file = Path(file)
 
-            if file.suffix.lower() in [".bz2", ".gz", ".z"]:
+            if file.suffix.lower() in (".bz2", ".gz", ".z"):
```

Comment:
My oversight, should use set instead `{".bz2", ".gz", ".z"}`

Reply:
Add a quick note for myself, `Sourcery` would recommend using `set` over `list` or `tuple` for membership check, see [here](https://docs.sourcery.ai/Coding-Assistant/Reference/Rules-and-In-Line-Suggestions/Python/Default-Rules/collection-into-set/). 

The advantage of `set/tuple` over `list` seems quite obvious, for example discussed [here](https://stackoverflow.com/questions/25368337/tuple-or-list-when-using-in-in-an-if-clause)

However I still don't quite know the difference between `tuple` and `set` in this scenario, [some](https://stackoverflow.com/questions/45017774/python-set-vs-tuple-lookup-is-lookup-in-tuple-o1) claims it's related to performance.